terraform/environments/staging/aws.tf

```diff
@@ -55,29 +55,29 @@ resource "aws_route" "private_nat_instance" {
   }
 }
 
-
 ################################################################################
-# Compute
+# EC2 Instance Connect Endpoint
 ################################################################################
 
-module "aws_bastion" {
-  source = "../../modules/aws/bastion"
-
-  ami  = data.aws_ami.ubuntu.id
-  name = "bastion - ${local.environment}"
-
-  associate_public_ip_address = true
-  instance_type               = "t3.micro"
-  key_name                    = aws_key_pair.staging.id
-  vpc_security_group_ids = [
-    module.sg_allow_all_egress.security_group_id,
-    module.sg_allow_ssh_ingress.security_group_id
```

Comment:
🤔 I don't think we could, because then port 22 would be blocked on the EC2 hosts at the AWS level.  That said, let me give it a try.

Reply:
If it does, I think you can update it to allow from the VPC CIDR (or wherever the instance connect endpoint is running "from") instead of 0.0.0.0/0 which will pass the check

Reply:
Actually, I just realized that we can probably get rid of that group.  I was thinking that group was on all the internal EC2 instances, but after looking I realized that this was only for the bastion host.  I should have looked before I commented originally, sorry about that.